components/constellation/constellation.rs

```diff
@@ -1594,10 +1594,18 @@ where
             top_level_browsing_context_id: top_level_browsing_context_id,
             browsing_context_id: browsing_context_id,
             new_pipeline_id: new_pipeline_id,
-            parent_pipeline_id: None,
             replace: None,
-            is_private: is_private,
-            is_visible: is_visible,
+            // TODO(mandreyel): is it possible for handle_panic to be invoked
+            // when there is no browsing context for pipeline? if not, we need
+            // not add this field
+            //
+            // If this browsing context didn't exist, insert a pending info
+            // entry so that we can construct it later.
+            new_browsing_context_info: Some(NewBrowsingContextInfo {
```

Comment:
This can be `None` since the `browsing_context_id` is from an already existing browsing context.